sound/soc/sof/sof-priv.h

```diff
@@ -30,6 +30,9 @@
 #define SOF_DBG_DYNAMIC_PIPELINES_ENABLE	BIT(4) /* 0: use static pipelines
 							* 1: use dynamic pipelines
 							*/
+#define SOF_DBG_DISABLE_D3_PERSISTENT	BIT(5)	/* disable D3 persistent feature
+						 * and FW reload needed
+						 */
```

Comment:
This is maybe details, but as we already have SBF_DBG_DYNAMIC_PIPELINES_ENABLE, it feels inconsistent to have DISABLE for this feature. I wonder if "IGNORE_D3_PERSISTENT" would be more appropriate as this feature depends on FW support in the end, but if driver ignores the bit, we always do a cold boot.

SOF_DBG_FORCE_COLD_BOOT is another option, but probably better use D3_PERSISTENT as that's the FW_READY flag, so it's obvious that the two are connected.

Just suggestions, I can live with this name. I do think the comment needs to be reworded a bit. Something like: 
```
/* ignore DSP D3 persistent capability\n and always cold boot */
```

Reply:
Um, it's not like the system 'cold boot' concept here, the IMR is still powered on during the DSP-reboot here even though the IMR restore feature is not used, but yes, "IGNORE_D3_PERSISTENT" sounds better to me.

Reply:
Ack @keyonjie , maybe drop the "cold boot" term, it's not really accurate. It's just a different boot flow.